openapi/components/schemas/Membership.yaml

```
@@ -9,10 +9,7 @@ properties:
     type: object
     properties:
       id:
-        description: Organization ID.
-        type: string
-        allOf:
-          - $ref: ./ResourceId.yaml
+        $ref: ./OrganizationId.yaml
       name:
         description: Organization name.
```

nitpick: good to add more detail here
```suggestion
        description: |-
             An organization is an entity that represents you, or your
             company, as a merchant. You can have multiple organizations.
             For more information, see
             [My organizations and websites](https://www.rebilly.com/docs/settings/organizations-and-websites/).
```